app/resources/forms.py

```diff
@@ -0,0 +1,39 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    IntegerField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    autocomplete = StringField('Enter the address')
+
+    name = StringField('Name', validators=[
+        InputRequired(),
+        Length(1, 64)
+    ])
+    description = StringField('Description', validators=[
+        InputRequired(),
+    ])
+    website = StringField('Website')
```

Comment:
I think we should be as strict as possibly if we are going to be taking text and probably putting them in `<a hrefs>` when we display them (I took 331 so I pretend I know about security things). Also, browsers automatically tack on the `http://` or `https://` when you copy and paste the whole url from the browser so I don't think it's that big of a deal.